internal/check-imports/src/imports.ts

```diff
@@ -0,0 +1,55 @@
+import * as abiCoder from '@fuel-ts/abi-coder';
+import * as abiTypegen from '@fuel-ts/abi-typegen';
+import * as address from '@fuel-ts/address';
+import * as asm from '@fuel-ts/asm';
+import * as contract from '@fuel-ts/contract';
+// forc-bin
+// fuels-gauge
+// fuels
```

Comment:
Did you forget to add these imports?

Reply:
Good catch! 🕵️

Yes and no:
 - The `fuels-gauge` and `forc-bin` do not export anything
 - My assumption for leaving the `fuels` out was a mistake — added back [here](https://github.com/FuelLabs/fuels-ts/pull/1055/commits/af8ec90ab0fc88019d1c4af2a9af7bc6f8bb91cc)

Thank you 🙏